packages/pangea-sdk/pom.xml

```diff
@@ -85,10 +85,11 @@
       <artifactId>log4j-core</artifactId>
       <version>2.20.0</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/commons-fileupload/commons-fileupload -->
     <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpmime</artifactId>
-      <version>4.5.14</version>
+        <groupId>commons-fileupload</groupId>
+        <artifactId>commons-fileupload</artifactId>
+        <version>1.5</version>
```

Comment:
nitpick: rest of the file is 2-space indent but this is introducing 4-space indent.

Reply:
We should add this file to pre-commit hooks. I'll check it.